File: python/moderation_video_token_demo.py
# -*- coding:utf-8 -*-
from ais_sdk.gettoken import get_token
from ais_sdk.moderation_video import moderation_video
from ais_sdk.utils import init_global_env

if __name__ == '__main__':
    #
    # access asr, long_sentence，post data by token
    #
    user_name = '******'
    password = '******'
    account_name = '******'  # the same as user_name in commonly use
    init_global_env(region='cn-north-1')

    demo_data_url = 'https://obs-test-llg.obs.cn-north-1.myhuaweicloud.com/bgm_recognition'
    token = get_token(user_name, password, account_name)

    # call interface use the url
    result = moderation_video(token, demo_data_url, 8)
    print result

File: javascript/moderation_video_token_demo.js
/**
 * 视频审核服务token方式请求的使用示例
 */
var video = require("./ais_sdk/moderation_video");
var token = require("./ais_sdk/gettoken");
var utils = require("./ais_sdk/utils");

// 初始化服务的区域信息，目前支持北京1(cn-north-1)、香港(ap-southeast-1)等区域信息
utils.initRegion(region="cn-north-1");

var username = "*******";        // 配置用户名
var domain_name = "*******";     // 配置用户名
var password = "*******";        // 密码

demo_data_url = "https://obs-test-llg.obs.cn-north-1.myhuaweicloud.com/bgm_recognition";

token.getToken(username, domain_name, password, function (token) {

    video.video(token, demo_data_url, 5, ["terrorism", "porn", "politics"], function (result) {
        console.log(result);
    })

});


